src/OrchardCore.Modules/OrchardCore.AutoSetup/Options/TenantSetupOptions.cs

```diff
@@ -75,6 +75,12 @@ public class TenantSetupOptions
         /// </summary>
         public string RequestUrlPrefix { get; set; }
 
+        /// <summary>
+        /// Gets or sets the name of the Feature Profile applied to the tenant. May be null or empty for no Profile
```

Comment:
/// Gets or sets the name of the feature profile applied to the tenant. May be <c>null</c> or empty if no profile
/// being selected.

Reply:
Please add these suggestions with this feature next time, it's more direct:

![image](https://user-images.githubusercontent.com/1976647/195411737-a2db2f12-9b84-4793-a830-4aff8a45a5b5.png)

Reply:
You are right